src/main/java/marquez/client/models/StreamMeta.java

```diff
@@ -45,48 +49,16 @@ public String toJson() {
     return Utils.toJson(this);
   }
 
-  public static Builder builder() {
```

Comment:
Removed this to make it consistent with using lombok and added additional setters which are not provided by lombok. Less code more cleaner.

Reply:
👍